tests/models.py

```diff
@@ -226,6 +237,14 @@ class ModelFour(db.Model):
     _model_twos = db.OneToMany(ModelTwo, backref=db.backref("_model_four"))
 
 
+class ModelFullCleanFail(db.Model):
+    pk = db.Column(db.Integer(), autoincrement=True, primary_key=True)
```

Comment:
might want to add field validators as well to make sure those validation errors update correctly